yivi-docs/blog/2025-02-08-ux/index.md

```diff
@@ -0,0 +1,104 @@
+# Yivi UX Plans
+One of the main goals for Yivi is to become the worlds most user friendly identity wallet.
+We've already done a lot to help this cause, like allowing the user to directly start issuing missing credentials while they're trying to dislose so they
+stay in the flow.
+
+In this blog we'll look ahead at some of the plans we have for the near future to improve the user experience.
+
+## Our four values for UX
+To keep our goals clear and keep an eye on the usefulness of new features, we've written down our main values for the UX.
+These values are:
+- Accessible
+- Understandable
+- Trustworthy
+- Beautiful
+
+Accessible means that we're committed to making Yivi a good experience for everyone.
+
+Understandable means that Yivi should feel familiar and usable to everyone.
+
+Trustworthy is not just about using the most reliable protocols. 
+It's also about conveying that to the user and letting them feel like they can trust Yivi, issuers and verifiers.
+Things that fall into this category are imagery, colors and clear explanations.
+
+Beautiful is not as important as the others, but the eye wants something too.
+Things like consistent color, fonts and styling combined with fluid animations is what gives Yivi just that extra touch.
+
+
+## Our plans for the near future
+There are four UX improvements that we want to work on over the next couple of months.
+These are:
+- Getting compliant with European Accessibility Act
+- QR code scanner button on the login page
+- Easier card organisation and navigation
+- Brand extensions
+
+Let's look at them one by one.
+
+### Getting compliant with European Accessibility Act (EAA)
+The EAA is a directive that aims to improve the functioning of the internal market for accessible products and services,
+by removing barriers created by divergent rules in member states of the European Union.
+
+It applies to products and services that have been identified as being most important to people with disabilities.
+Yivi aims to be one of those products and therefore should comply with this act.
+
+For digital products, like Yivi, this means having to comply with the Web Content Accessibility Guide (WCAG) version 2.1 level AA.
```

Comment:
An update in version 2.2 has recently been published. Shouldn't we strive to comply with that? https://www.w3.org/WAI/standards-guidelines/wcag/#versions